src/index.html

```diff
@@ -0,0 +1,79 @@
+<!DOCTYPE html>
+<html lang="en">
+<head>
+  <meta charset="UTF-8">
+  <meta name="viewport" content="width=device-width, initial-scale=1.0">
+  <title>Special Meal</title>
+
+  <link rel="stylesheet" href="./styles/style.css">
+  <link rel="stylesheet" href="./styles/popup.css">
+  <script src="https://kit.fontawesome.com/8d20eebc7a.js" crossorigin="anonymous"></script>
+</head>
+<body>
+
+  <header>
+    <div class="logo">
+      <img src="" alt="logo" id="logo">
+    </div>
+    <ul class="nav-links">
+      <a href="#">
+        <li class="Sea Food"><p>Sea Food <span id="count"></span> </li>
+      </a>
+      <a href="#">
+        <li class="Dessert"><p>Dessert</p></li>
+      </a>
+      <a href="#">
+        <li class="Vegetarian"><p>Vegetarian</p></li>
+      </a>
+    </ul>
+    
+  </header>
+
+  <main>
+    <section class="container" id="mealContainer">
+    </section>
+
+    <div id="popup" class="popup">
+      <div class="popup-content">
+        <div class="popup-heading">
+          <img id="popup-img" src="" alt="">
+          <button id="close-popup" class="close-popup-button close-button">
+            <i class="fa-solid fa-xmark"></i>
+          </button>
+        </div>
+        <div class="popup-description">
+          <h2 id="meal-title" class="meal-title"></h2>
+          <div class="instruction-container">
+            <p class="meal-instructions" id="meal-instructions"></p>
+          </div>
+        </div>
+        
+        <div class="comments-container">
+          <div class="comment-box">
+            <h3>
+              Comments <span id="comment-counter"></span>
+            </h3>
+            <ul id="comments-list" class="comment-list"></ul>
+          </div>
+
+          <form id="comment-form" class="comment-form">
+            <input id="comment-user" type="text" placeholder="Your name" class="comment-user" >
+            <textarea name="comment" id="comment-input" cols="30" rows="10" placeholder="Write a comment..." ></textarea>
+            <button type="submit" class="comment-button">Comment</button>
+          </form>
+        </div>
+      </div>
+    </div>  
```

Comment:
[OPTIONAL] I think it would be good for UX if you increase the height of the container for the comments and comment form. Some of the content appear squinted and we have to do a lot of scrolling for the comments

Reply:
Thank you very much for the suggestion @fobadara, we will implement the suggested changes.